src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -42,35 +42,55 @@
 	public static final String UNKNOWN_VIDEO_TYPEMIME = "video/mpeg";
 	public static final String UNKNOWN_IMAGE_TYPEMIME = "image/jpeg";
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
+	public static final String AUDIO_TRANSCODE = "audio/transcode";
+	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AC3_TYPEMIME = "audio/ac3";
+	public static final String AUDIO_ADPCM_TYPEMIME = "audio/x-adpcm";
 	public static final String AUDIO_ADTS_TYPEMIME = "audio/vnd.dlna.adts";
-	public static final String AUDIO_MP3_TYPEMIME = "audio/mpeg";
-	public static final String AUDIO_MP4_TYPEMIME = "audio/x-m4a";
-	public static final String AUDIO_WAV_TYPEMIME = "audio/x-wav";
-	public static final String AUDIO_WMA_TYPEMIME = "audio/x-ms-wma";
+	public static final String AUDIO_AIFF_TYPEMIME = "audio/aiff";
```

Comment:
@Nadahar audio/**x-**aiff should be prefered as more standard.
https://docs.oracle.com/cd/B28359_01/appdev.111/b28414/ap_audfmts.htm#BABCDCAD

While looking more at it, i found this about AIFC:
https://docs.oracle.com/cd/B28359_01/appdev.111/b28414/ap_audfmts.htm#BABIGBHG
It say that some AIFC can be compressed, but Wikipedia say that most of them are not.